.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .next
 .vscode
 dist
+cypress
```

Comment:
nit: newline

Reply:
Added the test file. I'm not familiar with what nit: newline means. Should there be an empty line at the end of every  file?